test/message_test.go

```diff
@@ -1104,12 +1105,16 @@ var _ = Describe("Remote Message Tests", func() {
 
 				publisher.PublishWithProperties(outboundMessage, resource.TopicOf(topic), config.MessagePropertyMap{key: value})
 
+				Eventually(func() uint64 {
```

Comment:
Maybe it's obvious to everyone else, but I'd love a comment here telling me whether Eventually blocks here or not.